inst/sympref.m

```diff
@@ -249,17 +230,6 @@
         settings.digits = arg;
       end
 
-    case 'snippet'
```

Comment:
Good point.  I wasn't going to bother with formal deprecation (keeping functionality for for one version) because its a debug thing, but I could at least warn as you suggest.